package.json

```diff
@@ -14,9 +14,9 @@
   "activationEvents": [
     "onCommand:mdb.connect",
     "onCommand:mdb.connectWithURI",
-    "onCommand:mdb.addConnection",
-    "onCommand:mdb.addConnectionWithURI",
+    "onCommand:mdb.disconnect",
     "onCommand:mdb.removeConnection",
+    "onCommand:mdb.openMongoShell",
```

Comment:
I think we don't use mongo without db anymore to not be accidentally offensive to someone. Maybe `openMongoDBShell ` would be better? Here we also call it `MongoDB Shell`: https://github.com/mongodb-js/mongosh